### ISSUE TITLE
More examples?

### ISSUE THREAD

**Issue by sotojuan** [redux-observable/redux-observable#35]
Hey, so as you know it'd be better to have more examples and I am willing to write some, just not sure which ones you want.

For example, [here](https://github.com/yelouafi/redux-saga/tree/master/examples) are the Redux Saga examples and [here](https://github.com/reactjs/redux/tree/master/examples) are the regular Redux examples.

Which ones would be beneficial to the community? I was going to do some of the counter ones but conceptually they'd be very similar to the basic example.

**Comment by jayphelps** [comment]
We're still feeling things out and the API is still in flux, so we're hesitant to spend a bunch of times writing examples that will quickly become out of date or a pain to keep updating. That indeed hints that users may want to steer clear until the API stabilizes.

The biggest change introduced is that we just added the ability to provide a function to the middleware initialization itself, where you can globally compose actions. This is much more like redux-saga, and helps with use cases like debounce/throttle. We suspect this will actually become the preferred method, replacing the existing "dispatch a thunkservable" way we currently have documented, but we don't want to update the docs only to change our mind yet again.

tl;dr bear with us, we're new to this too 😄

**Comment by istarkov** [comment]
@sotojuan processor example https://github.com/istarkov/redux-observable-process-fetch

**Comment by blesh** [comment]
Well, @robwormald set this up recently: https://plnkr.co/edit/8XCD02EoxkIfIWS4YAyA?p=preview

**Comment by madroneropaulo** [comment]
Any update on this? :)

**Comment by KevinGorjan** [comment]
+1

**Comment by jesinity** [comment]
I have ported the redux saga shopping cart example in redux observable, can I open a pull request?

**Comment by jayphelps** [comment]
@jesinity absolutely!

**Comment by hegelstad** [comment]
Real world use cases would be cool!

i.e. Validation and other business logic cases.

**Comment by gpolyn** [comment]
Also, see https://github.com/zeit/next.js/tree/canary/examples/with-redux-observable
